src/libraries/System.Runtime.Numerics/tests/BigInteger/GetBitLengthTests.cs

```diff
@@ -44,8 +44,8 @@ public static void RunGetBitLengthTests()
         public static void RunGetBitLengthTestsLarge()
         {
             // Very large cases
-            VerifyGetBitLength(BigInteger.One << 32 << int.MaxValue, int.MaxValue + 32L + 1, 1);
-            VerifyGetBitLength(BigInteger.One << 64 << int.MaxValue, int.MaxValue + 64L + 1, 1);
+            VerifyGetBitLength(BigInteger.One << 32 << (int.MaxValue / 2), (int.MaxValue / 2) + 32L + 1, 1);
+            VerifyGetBitLength(BigInteger.One << 64 << (int.MaxValue / 2), (int.MaxValue / 2) + 64L + 1, 1);
```

Comment:
We should add a comment clarrifying that the maximum number of bits is `(2^31) - 1`

and that we're just picking something that is reasonably large without forcing it up to the maximum

Reply:
Looks like we're still hitting an OOM with this which should be producing about a 128MB allocation.

It should likely be restricted to something that won't cause a reliable OOM

Reply:
Sure. Any preferences? This should be about 16MB

```suggestion
            // Values which are large but beneath the upper bound of
            // (2^31) - 1 bits and which should not cause OOM in CI.
            VerifyGetBitLength(BigInteger.One << 32 << (1 << 27), (1 << 27) + 32L + 1, 1);
            VerifyGetBitLength(BigInteger.One << 64 << (1 << 27), (1 << 27) + 64L + 1, 1);
```

Reply:
I think 16MB should be fine to test.